Allow PushBullet notifications to be sent to PushBullet channels (#11850)
netdata/netdata: health/notifications/pushbullet/README.md

1.  Signup and Login to pushbullet.com
2.  Get your Access Token, go to <https://www.pushbullet.com/#settings/account> and create a new one
3.  Fill in the PUSHBULLET_ACCESS_TOKEN with that value

> 4.  Add the recipient emails or channel tags (each channel tag must be prefixed with #, e.g. #channeltag) to DEFAULT_RECIPIENT_PUSHBULLET

Reviewer: ```suggestion
4.  Add the recipient emails or channel tags (each channel tag must be prefixed with #, e.g. #channeltag) to `DEFAULT_RECIPIENT_PUSHBULLET`.
```

>     !!PLEASE NOTE THAT IF AN EMAIL RECIPIENT DOES NOT HAVE A PUSHBULLET ACCOUNT, PUSHBULLET SERVICE WILL SEND AN EMAIL!!

Reviewer: ```suggestion
    > 🚨 The pushbullet notification service will send emails to the email recipient, regardless of if they have a pushbullet account.
```

> Set them in `/etc/netdata/health_alarm_notify.conf` (to edit it on your system run `/etc/netdata/edit-config health_alarm_notify.conf`), like this:

Reviewer: ```suggestion
To add notification channels, run `/etc/netdata/edit-config health_alarm_notify.conf` 

You can change the configuration like this:
```
